x-pack/plugins/security_solution/public/common/utils/resolve_path_variables.ts

```diff
@@ -10,5 +10,5 @@ export const resolvePathVariables = (
   variables: { [K: string]: string | number }
 ): string =>
   Object.keys(variables).reduce((acc, paramName) => {
-    return acc.replace(new RegExp(`\{${paramName}\}`, 'g'), String(variables[paramName]));
+    return acc.replace(new RegExp(`\\{${paramName}\\}`, 'g'), String(variables[paramName]));
```

Comment:
Yeah I was wondering that myself. It seems like the escaping doesn't really matter in this case. Either should either be double backslash or no backslashes at all. Do you prefer that I change it?

Reply:
I'm okay either way, but I suspect no backslashes would be easier to read and reason with. 🤔

Reply:
I guess the regex engine has a thing where if the curly braces doesn't contain a number, then it's treated as its literal. It's a bit too magic for my taste to be honest and I prefer to escape it so that a reader doesn't misinterpret it as a repetition quantifier 🤷 

Come to think it, we actually DO need to escape them for this exact reason. Because if `pathname` contains only numbers, and the curly braces are not escaped, the `RegExp` will throw an error.

Reply:
Oh, I see. It is because it is within a string literal is why we want to make sure that we escape the `{`. Let's go with that. 👍

Reply:
Yeah it's really confusing (but understandable when you know why) that the escaping rules are different between if you use the literal regex (`/foo/`) vs the `RegExp` constructor.

Reply:
Yeah! Typical JS 😆